credsweeper/__main__.py

```diff
@@ -59,6 +59,13 @@ def get_arguments() -> Namespace:
                         help="find files by predefined extension.",
                         dest="find_by_ext",
                         action="store_true")
+    parser.add_argument("--max_depth",
+                        help="recursive search in files which are zip archives.",
+                        type=positive_int,
+                        dest="max_depth",
+                        default=0,
+                        required=False,
+                        metavar="POSITIVE_INT")
```

Comment:
`max_depth` naming is good, but in my think it just used on zip files, so what about `zip_scan_depth` or `zip_depth` or so on..?
I think `max_depth` can be read scanning file depth..

Reply:
I suppose, the argument might be used for scan others containers like ISO images, tar-archives and so on.
Probably ```containers_depth``` or just ```depth```
Otherwise in future we have to change ```zip_``` prefix to avoid confuses.

Reply:
Checked `max_depth` has been changed to `depth` and resolve conversation.